changelogs/fragments/cve-2020-1736-revert.yml

```diff
@@ -0,0 +1,2 @@
+security_fixes:
+  - The fix for CVE-2020-1736 has been reverted and this CVE is no longer considered a flaw in ansible-base. Users are encouraged to specify a ``mode`` parameter in their file-based tasks when the files being manipulated contain sensitive data.
```

Comment:
I'm not sure this is accurate. I wouldn't be this definitive since there is still a race condition we need to address.

Reply:
```suggestion
  - The fix for CVE-2020-1736 has been reverted. Users are encouraged to specify a ``mode`` parameter in their file-based tasks when the files being manipulated contain sensitive data.
```